mtp_cashbook/apps/cashbook/tests/test_views.py

```diff
@@ -570,7 +584,8 @@ def test_new_credits_submit_with_uncreditable(self, *_):
             self.assertEqual(response.status_code, 200)
             self.assertContains(response, '1 credit sent to NOMIS')
             self.assertContains(response, '1 credit needs your manual input in NOMIS')
-            self.assertEqual(len(mail.outbox), 1)
+            self.assertEqual(len(mail.outbox), 0)
```

Comment:
Just to be sure I understand this correctly: `mail.outbox` had one element because it was using a Django email backend before - It has now 0 elements because emails are sent using an API call to Notify (we test the email was sent with the assertion below).

Reply:
it's probably a silly thing to check, but yeah the idea was to make sure no email was sent using the django email backend